pkg/controller/installation/installation_controller.go

```diff
@@ -251,7 +251,7 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 			logrus.Errorf("Error while copying alerting email address to RHMI CR: %v", err)
 		}
 	}
-
+	metrics.SetRateLimiting(20)
```

Comment:
this is a WIP: the value will be picked up from the config map created by: https://github.com/integr8ly/integreatly-operator/pull/1234